tutorials/dnssec-cloud-dns-domains/index.md

```diff
@@ -175,12 +175,12 @@ Confirm whether the TLD for your domain supports DNSSEC by looking for it in the
     for such a TLD. In any case, it is not recommended, since you should be
     aware (and in control) of any change in the DNSSEC status of your domain.
 
-If you  cannot reach the ICANN  TLD DNSSEC site  for any reason, check  with the
+If you  cannot reach the ICANN TLD DNSSEC site  for any reason, check with the
 [Verisign DNSSEC debugger][9]. Enter the TLD name and hit return; if you see any
-red ⊗ errors the TLD ([like AQ][10]) does not support DNSSEC .
+red ⊗ errors the TLD ([like AQ][10]) does not support DNSSEC.
 
-[9]: http://dnssec-debugger.verisignlabs.com/us
```

Comment:
The motivation for putting the US domain their was to provide an example; I felt the plain page (with the tiny little box in the upper left corner) was a bit opaque and hard to use. If you feel this change is less confusing, I'm fine with it, but wanted to explain the original intention behind it.

Reply:
I see your point, Alex, but I think that the instructions that you gave in the paragraph containing this link are adequate.